pyrs/dataobjects/fields.py

```diff
@@ -567,19 +567,21 @@ def __init__(self,
                  peak_tag: str = '',
                  hidraworkspace: Optional[HidraWorkspace] = None,
                  peak_collection: Optional[PeakCollection] = None,
-                 field_sample: Optional[ScalarFieldSample] = None) -> None:
+                 point_list: Optional[PointList] = None) -> None:
         r"""
         Converts a HidraWorkspace and PeakCollection into a ScalarField
         """
         self._peak_collection: Optional[PeakCollection] = None
-        # when the strain is composed of more than one scan, we keep references to them
-        self._single_scans: List['StrainField'] = []
-        self._field = field_sample
         self._filenames: List[str] = []
+        # when the strain is composed of more than one scan, we keep references to the individual ones
+        self._single_scans: List['StrainField'] = []
+        # the resolved ScalarFieldSample with the math done
+        self._field: Optional[ScalarFieldSample] = None
```

Comment:
`self._field` will eventually go away, right?

Reply:
Not in this PR. Small changes to prevent conflicts.

Reply:
Is there a follow-up issue? If so it'd be helpful to introduce a `# TODO #issue-number` to keep track